build/params_shared_funcs.go

```diff
@@ -39,14 +16,6 @@ func DhtProtocolName(netName dtypes.NetworkName) protocol.ID {
 	return protocol.ID("/fil/kad/" + string(netName))
 }
 
-func UseNewestNetwork() bool {
```

Comment:
Removed. It was only used by genesis, which now has the correct logic internally.